shared_model/validators/field_validator.cpp

```diff
@@ -102,6 +102,16 @@ namespace {
   const RegexValidator kAccountDetailKeyValidator{"DetailKey",
                                                   R"([A-Za-z0-9_]{1,64})"};
   const RegexValidator kRoleIdValidator{"RoleId", R"#([a-z_0-9]{1,32})#"};
+  const RegexValidator kPublicKeyHexValidator{
+      "PublicKeyHex",
+      fmt::format(
+          "[A-Fa-f0-9]{{{}}}",
+          shared_model::validation::FieldValidator::public_key_size * 2)};
+  const RegexValidator kSignatureHexValidator{
+      "SignatureHex",
+      fmt::format(
+          "[A-Fa-f0-9]{{{}}}",
+          shared_model::validation::FieldValidator::signature_size * 2)};
```

Comment:
Discussed offline to rework this later. My current idea is that it is more readable at the moment because the logic is contained at the same place in the code.